README.md

```diff
@@ -329,24 +329,18 @@ Fixed versions:
 
 Rolling versions:
 
-| Branch | Dockerfile     | Docker tag |
-| ------ | ----------     | ---------- |
-| master | 1.5/Dockerfile | 1.5        |
-| master | 1.6/Dockerfile | 1.6        |
-| master | 1.7/Dockerfile | 1.7        |
-| master | 1.9/Dockerfile | 1.9        |
-| master | 1.x/Dockerfile | 1          |
-| master | 1.x/Dockerfile | latest     |
-| master | 2.1/Dockerfile | 2.1        |
-| master | 2.2/Dockerfile | 2.2        |
-| master | 2.x/Dockerfile | 2          |
+| Branch | Dockerfile                | Docker tags |
+| ------ | ----------                | ----------- |
+| master | dockerfile/alpine_3.5_1.x | 1           |
+| master | dockerfile/alpine_3.5_2.2 | 2.1/2.2     |
+| master | dockerfile/alpine_3.5_2.x | 2           |
 
 Special builds:
 
-| Branch | Dockerfile             | Docker tag  |
-| ------ | ----------             | ----------  |
-| master | 1.x-centos7/Dockerfile | 1.x-centos7 |
-| master | 2.x-centos7/Dockerfile | 2.x-centos7 |
+| Branch | Dockerfile              | Docker tag  |
+| ------ | ----------              | ----------  |
+| master | dockerfile/centos_7_1.x | 1.x-centos7 |
```

Comment:
This patch is pushed - https://github.com/tarantool/docker/commit/a1fd398781cdc7a5eee63b78175dadf7612192cc

So, we can reuse one dockerfile for centos7-1.x and centos7-2.x

Reply:
It can't be done due to gitlab-ci job runs in parallel on the same hosts and logout for it's user may brake the logins from the other jobs.